File: despatch.WEB/app/src/swagger/models/contact-response.ts
/* tslint:disable */
import { BaseEntityResponse } from './base-entity-response';
import { ContactDataResponse } from './contact-data-response';
import { UserResponse } from './user-response';
export interface ContactResponse extends BaseEntityResponse{
  contactData?: ContactDataResponse;
  contactDataId?: string;
  friend?: UserResponse;
  friendId?: string;
  user?: UserResponse;
  userId?: string;
}

File: despatch.Domain/Models/Default/Response/ContactResponse.cs
namespace despatch.Domain.Models.Default.Response
{
    public class ContactResponse : BaseEntityResponse
    {
        public string UserId { get; set; }
        public string FriendId { get; set; }
        public string ContactDataId { get; set; }


        public virtual UserResponse User { get; set; }
        public virtual UserResponse Friend { get; set; }
        public ContactDataResponse ContactData { get; set; }
    }
}
